iiwa_cam/GeneralTraj general_traj
bool clean
---
bool success
string error_msg